actionview/lib/action_view/test_case.rb

```diff
@@ -42,6 +42,176 @@ def initialize
       end
     end
 
+    module Parsing
+      extend ActiveSupport::Concern
+
+      class Content < SimpleDelegator
+        def initialize(rendered, parsers)
+          super(rendered)
+          @parsers = parsers
+
+          @parsers.keys.each do |name|
+            instance_eval <<~RUBY, __FILE__, __LINE__ + 1
+              def self.#{name}
+                @parsers[#{name.inspect}].call(to_s)
+              end
+            RUBY
+          end
+        end
+      end
+
+      class_methods do
```

Comment:
I think methods defined inside this are not picked by rdoc? I think it is better to use `module ClassMethods`